crates/rolldown_plugin/src/plugin_driver/build_hooks.rs

```diff
@@ -194,8 +194,10 @@ impl PluginDriver {
     for (_, plugin, ctx) in
       self.iter_plugin_with_context_by_order(&self.order_by_transform_ast_meta)
     {
-      args.ast =
-        plugin.call_transform_ast(ctx, HookTransformAstArgs { cwd: args.cwd, ast: args.ast })?;
+      args.ast = plugin.call_transform_ast(
+        ctx,
+        HookTransformAstArgs { cwd: args.cwd, ast: args.ast, id: args.id },
```

Comment:
used to filter out `module` in `node_modules`, https://github.com/rolldown/vite/blob/8eed5766df862049aae43a7f1611acb83363b89a/packages/vite/src/node/plugins/importAnalysisBuild.ts#L217-L219